resources/user_config.ts

```diff
@@ -25,21 +30,55 @@ const ZoneId = _ZoneId;
 import _ZoneInfo from './zone_info';
 const ZoneInfo = _ZoneInfo;
 
+// Convince TypeScript and eslint that these are used.  TypeScript doesn't have a great way
```

Comment:
This feels odd to me; none of `disable-next-line`, `disable-line`, or `disable` are preferable here?

Reply:
I can do `/* eslint-disable @typescript-eslint/no-unused-vars */` but then there is still a `ts(6133)` warning in VSCode (or an error when running normally).  I can use a `// @ts-ignore` to ignore the next line, but that ignores everything.  There's (deliberately??) no way to disable a single warning for a single line.

So, I thought "let me just use these" and then we at least get all the other errors still, but suppress the "unused" one.